src/app/components/Select/index.tsx

```diff
@@ -20,7 +20,7 @@ const StyledButton = styled(Button)(({ theme }) => ({
   height: '36px',
   minWidth: '135px',
   padding: `0 ${theme.spacing(4)}`,
-  borderRadius: '12px',
+  borderRadius: '36px',
```

Comment:
![image](https://user-images.githubusercontent.com/9722540/222084268-c3b30db4-6516-41a6-9623-6132f50b1b5f.png)

So the reason why I didn't set the `border-radius` to the "correct" value according to the designs, is that the dropdown looks a bit off. Not having the same `border-radius` as the trigger button.

Reply:
dropdown list item hover state ? I will wait for Don's feedback in this case. thx for info

Reply:
Don wants to keep border-radius from designs.

Reply:
Right, would be nice to sync the dropdown container as well - I didn't found any good way of doing this.